presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -37,6 +37,16 @@ and ``join_type`` is one of
     FULL [ OUTER ] JOIN
     CROSS JOIN
 
+where ``grouping_element`` is one of
+
+.. code-block:: none
+
+    ()
+    expression
+    GROUPING SETS ( (column [, ...]) [, ...] )
```

Comment:
Are all of the `column` actually restricted to column? Can they be expressions or ordinals?